src/modules/blockchainApplication/explore/components/BlockchainApplicationStatistics/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { toRawLsk } from '@token/fungible/utils/lsk';
+import Box from 'src/theme/box';
+import BoxHeader from 'src/theme/box/header';
+import BoxContent from 'src/theme/box/content';
+import { DoughnutChart } from 'src/modules/common/components/charts';
+import TokenAmount from '@token/fungible/components/tokenAmount';
+import { tokenMap } from '@token/fungible/consts/tokens';
+import Tooltip from 'src/theme/Tooltip';
+import Icon from 'src/theme/Icon';
+import { useTheme } from 'src/theme/Theme';
+import { getColorPalette } from 'src/modules/common/components/charts/chartOptions';
+import styles from './blockchainApplicationStatistics.css';
+
+const BlockchainApplicationStatistics = ({ statistics }) => {
+  const { t } = useTranslation();
+  const colorPalette = getColorPalette(useTheme());
+
+  const doughnutChartData = {
+    labels: [
+      t('Registered'),
+      t('Active'),
+      t('Terminated'),
+    ],
+    datasets: [
+      {
+        backgroundColor: [colorPalette[1], colorPalette[0], colorPalette[2]],
+        data: [statistics.data.registered, statistics.data.active, statistics.data.terminated],
+      },
+    ],
+  };
+
+  const doughnutChartOptions = {
+    cutoutPercentage: 70,
+    legend: {
+      display: true,
+      position: 'left',
+      align: 'start',
+      labels: {
+        padding: 20,
+      },
+    },
+    layout: {
+      padding: {
+        left: 0,
+        right: 0,
+        bottom: 0,
+        top: 0,
+      },
+    },
+  };
```

Comment:
please move the part to utils and keep the component clean
```suggestion
  const {doughnutChartData, doughnutChartOptions} = prepareChartDataAndOptions(data, colorPalette)
```